src/gar.lua

```diff
@@ -705,6 +706,10 @@ function gar.cancelDelegateWithdrawal(from, gatewayAddress, vaultId)
 	delegate.delegatedStake = delegate.delegatedStake + vaultBalance
 	gateway.totalDelegatedStake = gateway.totalDelegatedStake + vaultBalance
 	GatewayRegistry[gatewayAddress] = gateway
+	return {
+		delegate = gar.getGateway(gatewayAddress).delegates[from],
+		totalDelegatedStake = gateway.totalDelegatedStake
+	}
```

Comment:
inconsequential change - but required a modification to the unit test that validates the result is no linger `nil`